sanowret-crystal.lic

```diff
@@ -54,6 +54,10 @@ class SanowretCrystal
       when /^You tap a .*sanowret crystal that you are holding\.$/
         use_crystal
         DRC.bput('stow my sanowret crystal', 'You put', 'Stow what')
+      when /Not here, /, /a wave of Corruption magic/
+        echo "Pausing #{Script.current.name} script until you move"
+        non_sano_room_id = Room.current.id
```

Comment:
💡 Should this be `@non_sano_room_id`?

💡 I'd also prefer it if you spelled out the full word `sanowret`